types/thing/Thing.md

```diff
@@ -0,0 +1,97 @@
+# Thing
+
+A thing is the most basic interface for any real or digital asset.
+
+## Schema
+
+The schema is defined by its Type committed on chain.
+
+
+## Example
+
+A [Type](../type/Type.md) is created with properties that look like this:
+
+```json
+{
+  "properties": [
+    {
+      "name": "title",
+      "type": "String",
+      "required": true
+    },
+    {
+      "name": "description",
+      "type": "md",
+      "required": false
+    }
+  ]
+}
+```
+
+We see that this Type has two properties; a title of type String and a description of type md (Markdown).
+
+When a Type is created, it also autogenerates basic Widgets for creating and displaying this data, and so our Type looks like:
```

Comment:
hey @elliotBraem, Do you mind to elaborate on this part `When a Type is created, it also autogenerates basic Widgets for creating and displaying this data`. What is this process of autogenerating components?